e2e/commands/patient-operations.ts

```diff
@@ -77,7 +77,7 @@ export const generateRandomPatient = async (api: APIRequestContext): Promise<Pat
         ],
         attributes: [],
         birthdate: '2020-2-1',
-        birthdateEstimated: true,
+        birthdateEstimated: false,
```

Comment:
Is this change necessary?

Reply:
Yeah. If we use `birthdayEstimated` as `true`, it will take the patient as a "patient with an unknown birthdate". So when we go to the edit patient page, it will automatically use the "Birthdate unknown" option. 

<img width="412" alt="image" src="https://github.com/openmrs/openmrs-esm-patient-management/assets/27498587/10f02014-709f-4adc-8776-ebbe5827b136">

(That change was added with 7dbf922e29a96925edb0bc700b4383f904b8ac6a. And the tests were failing after that)

Reply:
Yeah, that's right. I changed the date format.